packages/components/icon/stories/Icon.stories.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { Fragment } from 'react';
+import { Flex } from '@contentful/f36-core';
+import { Paragraph, SectionHeading } from '@contentful/f36-typography';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { MdAcUnit as ExternalIcon } from 'react-icons/md';
+
+import { Icon } from '../src/';
+import type { IconProps } from '../src/';
+
+export default {
+  argTypes: {
+    as: { control: { disable: true } },
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    testId: { control: { disable: true } },
+  },
+  component: Icon,
+  parameters: {
+    propTypes: [Icon['__docgenInfo']],
+  },
+  title: 'Components/Icon',
+} as Meta;
+
+export const WithSVGPath: Story<IconProps & { as?: never }> = (args) => (
+  <Fragment>
+    <SectionHeading as="h3" marginBottom="spacingS">
+      Icon component with SVG paths
+    </SectionHeading>
+
+    <Icon {...args}>
```

Comment:
@denkristoffer I imagine it's a bit late in the process, but I am wondering about the usage of custom icons. It seems more intuitive to me to use a custom icon somehow like this in the component `<Icon as={MyCustomIcon} size="small" color"whatever" />`, not really passing paths as children. 🤔

Reply:
Right now I would need to copy and paste paths inside the `<Icon></Icon>` as children. Am I right?

Reply:
You can do it both ways 😄

Reply:
@denkristoffer ok, then I would extend the REAMDE file to clearly show how custom icons might be used both ways. I must say, that this flow seems not very comfortable to me - I need to open my custom SVG file, copy and paste paths inside the <Icon> component. As a user, I would probably just do my custom SVG instead.🤷‍♀️

Reply:
What kind of flow do you think users would like? Can we make it more comfortable?

Reply:
I am assuming here. For me it would be something like:

```
import myCustomIcon from 'custom-icon.svg'

<Icon as={myCustomIcon} size="small" />

or
<Icon size="small">{myCustomIcon}</Icon>

```
To me, it's a bit odd to copy and paste paths inside the `<Icon>` component. But maybe it's just me. If you can do it both ways, that's great. Let's point it out in the README - I mean, info about `as={}` option is there, but for me, it seems like we recommend using paths when you use a custom icon. It's just wording or my interpretation of it. 😁  It might be just me.
It's not a blocker. 🙂